docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

```diff
@@ -235,71 +261,38 @@ While `quarkus-oidc` and `quarkus-elytron-security-oauth2` can also verify such
 
 [NOTE]
 ====
-Your decision to choose whether to use opaque or JWT token format will be driven by architectural considerations.
+Architectural considerations drive your decision to use opaque or JWT token format.
 Opaque tokens tend to be much shorter than JWT tokens but need most of the token-associated state to be maintained in the provider database.
 Opaque tokens are effectively database pointers.
-JWT tokens are significantly longer than the opaque tokens but the providers are effectively delegating storing most of the token-associated state to the client by storing it as the token claims and either signing or encrypting them.
+JWT tokens are significantly longer than opaque tokens. Still, the providers effectively delegate most of the token-associated state to the client by storing it as the token claims and either signing or encrypting them.
 ====
 
-The following table provides a summary of the options for each authentication mechanism:
-
+[[table]]
+.Token authentication mechanism comparison
 |===
-| | quarkus-oidc| quarkus-smallrye-jwt | quarkus-elytron-security-oauth2
-
-|Requires Bearer JWT verification
-|Local verification or introspection
-|Local verification
-|Introspection
-|Requires Bearer opaque token verification
-|Introspection
-|No
-|Introspection
-|Refreshing `JsonWebKey` set for verifying JWT tokens
-|Yes
-|Yes
-|No
-|Represent token as `Principal`
-|Yes
-|Yes
-|Yes
-|Inject JWT as MP JSON Web Token (JWT)
-|Yes
-|Yes
-|No
-|Authorization Code Flow
-|Yes
-|No
-|No
-|Multi-tenancy
-|Yes
-|No
-|No
-|User info support
-|Yes
-|No
-|No
-|PEM key format support
-|No
-|Yes
-|No
-|SecretKey support
-|No
-|In JSON Web Key (JWK) format
-|No
-|Inner-signed and encrypted or encrypted tokens
-|Introspection
-|Local verification
-|Introspection
-|Custom token verification
-|No
-|With injected JWT parser
-|No
-|Accept JWT as a cookie
-|No
-|Yes
-|No
+^|Feature required 3+^| Authentication mechanism
+
+^| ^s|`quarkus-oidc` ^s|`quarkus-smallrye-jwt` ^s| `quarkus-elytron-security-oauth2`
+
+s|Bearer JWT verification ^|Local verification or introspection  ^|Local verification  ^|Introspection
+
+s|Bearer opaque token verification ^|Introspection   ^|No  ^|Introspection
+s|Refreshing `JsonWebKey` set for verifying JWT tokens  ^|Yes   ^|Yes  ^|No
+s|Represent token as `Principal`  ^|Yes   ^|Yes  ^|Yes
+s|Inject JWT as MP JSON Web Token (JWT)  ^|Yes   ^|Yes  ^|No
```

Comment:
Style/Capitalization: 
- Use sentence case - "Inject JWT as MP JSON web token 
- Is it necessary to have (JWT) written in brackets in this instance given that it is already used in a few previous instances in the table. Would it be possible to instead place it after the very first occurrence in the topic?

Reply:
`JSON Web Token` is how it is used on the web/everywhere, it is OK IMHO

Reply:
Will do. Thx for the catch